src/App.scss

```diff
@@ -11,29 +13,11 @@ button {
   cursor: pointer;
 }
 
-.error {
-  color: red;
+.container:not(.my-container) {
+  width: 400px;
+  margin: 30px;
```

Comment:
Consider adding a margin to the top and bottom of the container instead of just the top. 📏